docs/en/ExtraFiles.md

```diff
@@ -0,0 +1,104 @@
+# Adding extraData into distributive.
+
+CQtDeployer has support to work with extra data files. 
+
+For this you should use the -extraData and -extraDataOut options. 
+
+## Examples:
+
+### Add custom file into root directory of the your distribution.
+
+```
+cqtdeployer -extraData path/to/my/customFile
+```
+
+Or in configFile.json
+
+```json
+{
+    ...
+    "extraData": "./path/to/my/customFile"
+    ...
+}
+```
+
+This command support list of arrguments.
+
+
+``` bash
+cqtdeployer -extraData path/to/my/customFile,path/to/my/customFile2,path/to/my/customFile3
+```
+
+Or in configFile.json
+
+```json
+{
+    ...
+    "extraData": [
+        "./path/to/my/customFile",
+        "./path/to/my/customFile2",
+        "./path/to/my/customFile3"
+    ]
+    ...
+}
+```
+
+### Changing location of extra data files.
+
+For change location you should use  the extraDataOut options. This option change destination path relative the root of the distributive.
```

Comment:
This option change destination path based on the root of the distributive

Reply:
Why do you change relative to base? it changes the meaning.